shared-ts/jsonTypes.ts

```diff
@@ -212,16 +212,21 @@ export type Source = {
 };
 
 export type Period = {
```

Comment:
Just noticed the filename is being changed back here, I'd leave it as index.d.ts since it's the more standard way to declare a type file

Reply:
done